components/Iframe.tsx

```diff
@@ -6,6 +6,6 @@ type IframeProps = { src: string } & HTMLAttributes<HTMLIFrameElement>;
 
 export default function Iframe(props: IframeProps) {
   return (
-    <iframe width="100%" height="500px" style={{ border: "none" }} {...props} />
+    <iframe width="100%" height="500px" {...props} />
```

Comment:
Just curious about the border spec. I'd expect you _dont_ want a border on these iframes.

Is this style just not needed?

Reply:
Not needed